gef.py

```diff
@@ -621,18 +621,99 @@ def __str__(self):
     def is_valid(self):
         return "(bad)" not in self.mnemonic
 
+class MallocStateStruct(object):
+    def __init__(self, addr):
+        self.addr = addr
+        self.num_fastbins = 11 if is_x86_32() else 10
+        self.num_bins = 254
+        self.size_t = cached_lookup_type("size_t")
+        if not self.size_t:
+            ptr_type = "unsigned long" if current_arch.ptrsize == 8 else "unsigned int"
+            self.size_t = cached_lookup_type(ptr_type)
+
+    @property
+    def fastbins_addr(self):
+        return self.addr + 8 + current_arch.ptrsize
```

Comment:
are these +8 and +16 the same even on 32 bit?

Reply:
yeah its two `ints` which are both size 4. I'll grab it from the type though instead